src/python/dxpy/bindings/apollo/dataset.py

```diff
@@ -0,0 +1,120 @@
+from dxpy import DXHTTPRequest
+from dxpy.bindings import DXRecord
+
+
+class Dataset(DXRecord):
+    def __init__(self, record_id, project_id):
+        self.record_id = record_id
+        self.project_id = project_id
+        self.visualize_info = None
+        self._dx_record_obj = None
+        self.dx_dataset_descriptor = None
+        self.assays_list = None
+        self.assay_names_list = None
+
+    def get_visualize_info(self):
+        if self.visualize_info is None:
+            self.visualize_info = DXHTTPRequest(
+                "/" + self.record_id + "/visualize",
+                {"project": self.project_id, "cohortBrowser": False},
+            )
+        return self.visualize_info
```

Comment:
This is good, you could also improve this for downstream use and make it a property, while storing the info in a "private" attribute `self._visualize_info`:
```
@property
def visualize_info(self):
    if self._visualize_info is None:
       ...
    return self._visualize_info
```
So that later on, you could use it like this:
```
self.visualize_info["url"]
```